ibm_catalog.json

```diff
@@ -0,0 +1,163 @@
+﻿{
+  "products": [
+    {
+      "label": "deploy-arch-ibm-wazi-vsi",
+      "name": "Wazi as a service VSI on VPC Landing zone",
+      "product_kind": "solution",
+      "tags": [
+        "virtualservers",
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "reference_architecture",
+        "solution"
+      ],
+      "keywords": [
+        "vpc",
+        "slz",
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution",
+        "vsi",
+        "virtual server instance"
+      ],
+      "short_description": "Creates a secure infrastructure with virtual servers to run your workloads on a VPC network",
+      "long_description": "The VSI on VPC landing zone provides secure and customizable compute resources for running your applications and services. It creates secure and compliant Virtual Server Instances (VSI) on top of an existing Virtual Private Cloud (VPC) network.",
+      "offering_icon_url": "https://globalcatalog.cloud.ibm.com/api/v1/1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc/artifacts/solution.svg",
+      "provider_name": "IBM",
+      "flavors": [
+        {
+          "label": "Standard",
+          "name": "standard",
+          "usage": "Add an example of the use of the module in the following code block.\n\nUse real values instead of \"var.\\u003cvar_name\\u003e\" or other placeholder values\nunless real values don't help users know what to change.\n--\\u003e",
+          "working_directory": "solutions/standard",
+          "licenses": [
+            {
+              "id": "LICENSE",
+              "name": "LICENSE",
+              "type": "text/plain",
+              "description": "LICENSE"
+            }
```

Comment:
hmm @kbiegert not sure an opensource license should be included for an IBM Cloud DA in catalog? Any guidance here?